src/project/mod.rs

```diff
@@ -870,7 +389,7 @@ impl Project {
 
         // Gather the main activation scripts if there are no target scripts defined.
         if all_scripts.is_empty() {
-            if let Some(activation) = &self.manifest.activation {
+            if let Some(activation) = &self.manifest.manifest.activation {
```

Comment:
the manifest.manifest duplication looks a bit funky. can we rename one of them?

Reply:
it's now `manifest.parsed`